mobify/platform-scaffold
# Combine lighthouse-ci and lighthouse-local into lighthouse-preview

One less script to worry about 🎉 

## Changes
- `npm run test:pwa-preview` runs Lighthouse with Preview, and this works locally (`npm run dev` in another tab) and on CI (test server running in background)

## How to test-drive this PR
- Rebuild on CI https://circleci.com/gh/mobify/platform-scaffold/tree/combine-lighthouse-scripts
- Or run locally: `npm run dev`, `npm run test:pwa-preview` in another tab
- Verify that Lighthouse report is generated, and the score is checked
